src/rest_parser.cpp

```diff
@@ -618,6 +618,9 @@ Status KFSRestParser::parseInput(rapidjson::Value& node, bool onlyOneInput) {
         if (!dim.IsInt()) {
             return StatusCode::REST_COULD_NOT_PARSE_INPUT;
         }
+        if (dim.GetInt() <= 0) {
```

Comment:
Please add check in model instance before model reload to check status from request validation.
We should only extract request shape & batch size only if validation returns: StatusCode::OK, reshape_requred, batchsize_change_required

Reply:
done